include/schrodinger/rdkit_extensions/convert.h

```diff
@@ -26,6 +26,23 @@ namespace schrodinger
 namespace rdkit_extensions
 {
 
+const std::vector<Format> AUTO_DETECT_FORMATS = {
+    Format::RDMOL_BINARY_BASE64,
+    Format::MDL_MOLV3000,
+    Format::MAESTRO,
+    Format::INCHI,
+    Format::PDB,
+    Format::MOL2,
+    Format::XYZ,
+    Format::MRV,
+    Format::CDXML,
+    // Attempt SMILES before SMARTS, given not all SMARTS are SMILES
+    Format::SMILES,
+    Format::SMARTS,
+    // Guess at HELM after guessing atomistic formats
+    Format::HELM,
```

Comment:
One last question! Would it be reasonable to update this to:
```
// Guess at monomer formats after guessing atomistic formats
 Format::HELM,
// Assume peptide FASTA since there's no way to guess otherwise
Format::FASTA_PEPTIDE,
```
or is that silly?